### DISCUSSION TITLE
Collab not creating Fast-Dreambooth folder on Google drive?

### DISCUSSION THREAD

**Discussion by merper** [TheLastBen/fast-stable-diffusion#1970] [Q&A]
Probably a dumb question, but I've run the Import Drive and Dependencies model on the collab (linked to my G Drive) and yet I don't see a folder created in my Google drive yet to drop my HuggingFace Token into. Any quick fix for this?

**Comment by TheLastBen** [comment]
run the session to create the folder

**Reply by merper** [reply]
simple fix. Worked. Thanks!
